subsys/dfu/img_util/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_sources_ifdef(CONFIG_MCUBOOT_IMG_MANAGER flash_img.c)
+
+zephyr_library_link_libraries(MCUBOOT_BOOTUTIL)
```

Comment:
is "img_util" supposed to always depend on mcuboot?

Reply:
Currently it is enabled by the same Kconfig property as dfu/boot. So yes for now.
Although it can be decoupled easily in feature.